citesphere/src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml

```diff
@@ -14,14 +17,30 @@
 
 	<!-- Handles HTTP GET requests for /resources/** by efficiently serving 
 		up static resources in the ${webappRoot}/resources directory -->
-	<mvc:resources mapping="/resources/**" location="/resources/" />
-    <mvc:annotation-driven/>
+	<mvc:resources mapping="/resources/**"
+		location="/resources/" />
+	<mvc:annotation-driven />
 	<se:global-method-security
 		pre-post-annotations="enabled" />
-        
-        
-    <context:component-scan base-package="edu.asu.diging.citesphere.web, edu.asu.diging.citesphere.api, edu.asu.diging.citesphere.config.web"></context:component-scan>
-		
+
+	<beans:bean
+		class="org.springframework.data.web.config.SpringDataWebConfiguration" />
+	<mvc:annotation-driven>
+		<mvc:argument-resolvers>
+			<ref bean="sortResolver" />
+			<ref bean="pageableResolver" />
+		</mvc:argument-resolvers>
+	</mvc:annotation-driven>
+
+	<bean id="sortResolver"
+		class="org.springframework.data.web.SortHandlerMethodArgumentResolver" />
+	<bean id="pageableResolver"
+		class="org.springframework.data.web.PageableHandlerMethodArgumentResolver">
+		<constructor-arg ref="sortResolver" />
+	</bean>
+	<context:component-scan
```

Comment:
the component scan should be back at the beginning of the page, since that's where we usually put it and makes it super confusing if it's not there anymore.